src/app/competency-passbook/competency-card-details/competency-card-details.component.html

```diff
@@ -110,13 +110,13 @@
                             </div>
                         </div>
                     </ng-container>
-                    <div class="flex items-center text-primary underline cursor-pointer text-xs" *ngIf="themeDetails.courseSubThemes[i]?.length > 2 && !eachCert.viewMore" (click)="handleViewMore(eachCert)">
+                    <!-- <div class="flex items-center text-primary underline cursor-pointer text-xs" *ngIf="themeDetails.courseSubThemes[i]?.length > 2 && !eachCert.viewMore" (click)="handleViewMore(eachCert)">
                         {{'competencyCardDetails.viewMore' | translate}}
                     </div>
                     <div class="flex items-center text-primary underline cursor-pointer text-xs" *ngIf="eachCert?.viewMore"
```

Comment:
Please Revise this code, its wrong

Reply:
Me & Venkat have added only the translate pipe here, so the code outside may have come during merging the branches. So removing the duplicate code outside the ng-container should be fine.